lib/scaleway/bootstrap/ks-object-bucket.tf

```diff
@@ -0,0 +1,11 @@
+# TODO(benjaminch): To be handled directly in the engine to properly handle reinstall
+resource "scaleway_object_bucket" "object-bucket" {
+  # NOTE: name supports only alpha-numerics!
+  name = "test-cluster" # TODO(benjaminch): use var.s3_bucket_kubeconfig
+  acl  = "private"
+  versioning {
+    enabled = true
+  }
+
+  tags = local.tags_ks
```

Comment:
same for tags

Reply:
This is ok I guess, it's already in a var and support plain dictionary.